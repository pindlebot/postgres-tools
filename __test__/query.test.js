require('dotenv').config()

const query = require('../src/query')

// it ('query', () => {
//   return query('INSERT INTO users(id, name) values($1::int, $2::text) returning *', [2, 'lorem'])
//     .then(rows => {
//       return expect(rows[0]).toMatchSnapshot()
//     })
// })

it ('SELECT * FROM users WHERE id = $1', () => {
   return query('SELECT * FROM users WHERE id = $1', [2])
    .then(rows => {
      return expect(rows[0]).toMatchSnapshot()
    })
})
