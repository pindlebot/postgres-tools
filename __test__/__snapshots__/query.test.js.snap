// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SELECT * FROM users WHERE id = $1 1`] = `
Object {
  "id": 2,
  "name": "lorem",
}
`;

exports[`query 1`] = `
Object {
  "id": 2,
  "name": "lorem",
}
`;
